test/unit/view_model/active_record/cache_test.rb

```diff
@@ -147,8 +147,15 @@ module BehavesLikeACache
             value(view_name).must_be(:present?)
```

Comment:
If you're doing misc linter cleanups, apparently this file has unnecessary directives:
```
diff --git a/test/unit/view_model/active_record/cache_test.rb b/test/unit/view_model/active_record/cache_test.rb
index 85bf8a8..ec05e7d 100644
--- a/test/unit/view_model/active_record/cache_test.rb
+++ b/test/unit/view_model/active_record/cache_test.rb
@@ -1,7 +1,5 @@
 # frozen_string_literal: true
 
-# rubocop:disable Style/RedundantParentheses, Lint/MissingCopEnableDirective
-
 require "minitest/autorun"
 require "minitest/unit"
 require "minitest/hooks"
```